### ISSUE TITLE
Draggable's `endDrag` type declaration requires the `event` param

### ISSUE THREAD

**Issue by lwchadderdon** [greensock/GSAP#506]
The current [type definition for endDrag](https://github.com/greensock/GSAP/blob/4d2d07bfd5d5890e3aa1a13f0eb16a7758b02f2e/types/draggable.d.ts#L191) requires the `event` param:
```typescript
/**
 * Force the Draggable to immediately stop interactively dragging. 
 * You must pass it the original mouse or touch event that initiated the stop.
 *
 * ```js
 * draggable.endDrag(e);
 * ```
 *
 * @param {Event} event
 * @memberof Draggable
 * @link https://greensock.com/docs/v3/Plugins/Draggable/endDrag()
 */
endDrag(event: Event): void;
```

Based on the [implementation](https://github.com/greensock/GSAP/blob/4d2d07bfd5d5890e3aa1a13f0eb16a7758b02f2e/src/Draggable.js#L1624) though, the event param is optional and defaults to the Draggable's `pointerEvent`:
```javascript
this.endDrag = e =>	onRelease(e || self.pointerEvent, true);
```

The documentation does state that the `event` param is required:
https://greensock.com/docs/v3/Plugins/Draggable/endDrag()
so I'm not sure what the intent is, but it seems to function fine without passing an event in so I think a safe change would be to update the type definition to make the event param optional:
```typescript
/**
 * ...
 * @param {Event} [event]
 * @memberof Draggable
 * @link https://greensock.com/docs/v3/Plugins/Draggable/endDrag()
 */
endDrag(event?: Event): void;
```

**Comment by jackdoyle** [comment]
Yeah, it technically works in most cases but that assumes there has been a valid event at some point that populates the Draggable's .pointerEvent property. That's almost always a valid assumption but there could be an edge case where that wouldn't be valid. Like if you call .endDrag() on a Draggable that has never even started dragging or been pressed. See what I mean? 

If you're trying to avoid a TypeScript error and you don't want to pass in an event, you could just pass in the Draggable's .pointerEvent.
